segmentstore/storage/impl/src/main/java/io/pravega/segmentstore/storage/impl/rocksdb/RocksDBConfig.java

```diff
@@ -25,6 +25,7 @@
     public static final Property<Integer> WRITE_BUFFER_SIZE_MB = Property.named("writeBufferSizeMB", 64);
     public static final Property<Integer> READ_CACHE_SIZE_MB = Property.named("readCacheSizeMB", 8);
     public static final Property<Integer> CACHE_BLOCK_SIZE_KB = Property.named("cacheBlockSizeKB", 32);
+    public static final Property<Boolean> DIRECT_READS = Property.named("directReads", true);
```

Comment:
This is different than `master`. We should make them the same.

https://github.com/pravega/pravega/blob/master/segmentstore/storage/impl/src/main/java/io/pravega/segmentstore/storage/impl/rocksdb/RocksDBConfig.java#L28

Reply:
We want to keep the behavior the same as currently have in `0.5`, that's the reason we changing the default, so that we match the current behavior.

Reply:
@fpj told me offline he requested this. So I withdraw my comment.

Reply:
For completeness, I have requested it this way because we did a good amount of testing with this behavior, and I'd rather not change at this point. For `master`, we have a different default value.